### DISCUSSION TITLE
2006/09/22/%E6%AD%A3%E5%B8%B8%E8%AE%BF%E9%97%AEgoogle%E7%BD%91%E9%A1%B5%E5%BF%AB%E7%85%A7%E7%9A%84%E6%96%B9%E6%B3%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#116] [Announcements]
# 2006/09/22/%E6%AD%A3%E5%B8%B8%E8%AE%BF%E9%97%AEgoogle%E7%BD%91%E9%A1%B5%E5%BF%AB%E7%85%A7%E7%9A%84%E6%96%B9%E6%B3%95

转载的，原文在：http://www.williamlong.info/archives/634.html

https://blog.qizhen.xyz/2006/09/22/%E6%AD%A3%E5%B8%B8%E8%AE%BF%E9%97%AEgoogle%E7%BD%91%E9%A1%B5%E5%BF%AB%E7%85%A7%E7%9A%84%E6%96%B9%E6%B3%95

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Qizhen
2006年09月26日 21:02 编辑
我现在也还没有完全治好，也没有做过过敏原测试。
基本上持续的体育锻炼是体高免疫力的唯一办法了，要锻炼多长时间使因人而异的，
但至少也的一年半载。
只要坚持，肯定会有好处的。
 
看了上面这番回答，你可能会觉得有些失望。
作为一个人工荨麻疹的老病号，我是能理解你的痛苦，和机遇摆脱病痛的心情的。
我自己也研究这病好久了。
人工荨麻疹现在也没有找到它的确切发病原因，更没有特效药。
治愈它是需要时间和坚持不懈的努力的。
所以在现在这个阶段调整好自己心情是最重要的了。
比人工荨麻疹更痛苦疾病多的是，跟得了那些疾病的人相比，我们还算是幸运的了。

回复
斯
2006年09月26日 16:27 编辑
嘿，我在网上搜到你的sp，我的过敏症状跟你的是一样的，想问一下你现在治好了么？用了什么方法阿？你做过过敏源测试没？运动提高免疫力真的有效么？大概要运动多久才生效阿？我被折磨得快死了。。 http://angeline.blog.sohu.com

回复
毅
2006年09月25日 15:25 编辑
简单明了
